### ISSUE TITLE
Notifications disappear

### ISSUE THREAD

**Issue by LitoFrame** [Ranchero-Software/NetNewsWire#2786]
This was true for all 6.0 versions. Now with 6.0a1 I can close the notifications without open the article, but approx. 1 min all the notification disappear even I don't touch them. I have Alerts notifications in system settings. 
In NNW 5 the notifications didn't disappear.
macOS Big Sur

**Comment by vincode-io** [comment]
It looks like your system thinks that you have selected Banners instead of Alerts in the System Settings.  Could you toggle it to Banners then back to Alerts and to see if it helps?

**Comment by vincode-io** [comment]
When I have Alerts set in System Settings the notification doesn't disappear for me.  I left it up there for almost 1/2 hour.

<img width="358" alt="Screen Shot 2021-02-05 at 3 54 32 PM" src="https://user-images.githubusercontent.com/16448027/107093256-8c4f6400-67ca-11eb-9440-c8de79f6aae6.png">

**Comment by LitoFrame** [comment]
It is very strange for me, because when I set banner notification in system pref I have no notification at all, but when I set it to alerts it is like banner...

**Comment by LitoFrame** [comment]
I had no issue with NNW 5 and other notifications...

**Comment by kevinm6** [comment]
Sorry @LitoFrame , can you just share a screenshot of the notification settings of NNW ??
I never had similar problems

**Comment by LitoFrame** [comment]
![Screenshot 2021-02-06 at 9 53 06](https://user-images.githubusercontent.com/42410142/107114970-5b6e3e00-6861-11eb-872c-ec1bcfd918de.png)

**Comment by kevinm6** [comment]
Weird. 
I can just tell you to try to quit NNW, then go here in Notifications Settings, delete the NNW voice, quit Settings, open again NNW and allow again the permission for notifications.

**Comment by LitoFrame** [comment]
Unfortunately it didn't help...

**Comment by LitoFrame** [comment]
@kevinm6 I did again your suggestion, but now before I opened NNW again I restarted my computer. Things are improved, but still not good. The notifications stay longer, but disappear latest the next refresh. It seems that On My Mac account is more "stable", than iCloud.
When I click Mark All as Reed, the notifications disappear (which I think is the right behaviour). Is it possible, that somehow NNW "assumes" that I read that article?
I will do further testing later...

**Comment by LitoFrame** [comment]
I used https://www.flickr.com/services/feeds/photos_public.gne for my test. When I add it to On My Mac account the notifications stayed almost every time until refresh. Manually or automatically doesn't matter, after refresh I got new notifications. I had 1 or 2 occasion when 1 notification disappeared. 
When I add it to iCloud, the notification stayed approx 1 or 2 mins or until I manually refresh the feed.
